Libraries/LibC/ctype.h

```diff
@@ -57,18 +57,6 @@ int isgraph(int);
 int islower(int);
 int isupper(int);
 
-#define isalnum(c) (_ctype_[(unsigned char)(c)] & (_U | _L | _N))
-#define isalpha(c) (_ctype_[(unsigned char)(c)] & (_U | _L))
-#define iscntrl(c) (_ctype_[(unsigned char)(c)] & (_C))
-#define isdigit(c) (_ctype_[(unsigned char)(c)] & (_N))
-#define isxdigit(c) (_ctype_[(unsigned char)(c)] & (_N | _X))
-#define isspace(c) (_ctype_[(unsigned char)(c)] & (_S))
-#define ispunct(c) (_ctype_[(unsigned char)(c)] & (_P))
-#define isprint(c) (_ctype_[(unsigned char)(c)] & (_P | _U | _L | _N | _B))
-#define isgraph(c) (_ctype_[(unsigned char)(c)] & (_P | _U | _L | _N))
-#define islower(c) ((_ctype_[(unsigned char)(c)] & (_U | _L)) == _L)
-#define isupper(c) ((_ctype_[(unsigned char)(c)] & (_U | _L)) == _U)
```

Comment:
Could we at least have them as `static inline` functions in the header? It'd be sad to lose inlining for them.

Reply:
I tried doing this. While I thought it worked at first, it actually fails with undefined references when I rebuilt. Here's my ctype.h: https://gist.github.com/Lubrsi/17b4db531338f6a53692086a47cbbd59
And the log: https://gist.github.com/Lubrsi/e6912f26216949a93a8f5f54f119885d
Put that on the completely wrong comment 😄

Reply:
Nevermind, I did a clean rebuild today and it worked.